src/main/kotlin/build/buf/intellij/annotator/BufAnalyzeModificationTracker.kt

```diff
@@ -25,17 +26,19 @@ import com.intellij.protobuf.lang.PbLanguage
 import com.intellij.psi.search.FilenameIndex
 import com.intellij.psi.search.GlobalSearchScope
 import com.intellij.psi.util.PsiModificationTracker
+import io.kanro.idea.plugin.protobuf.lang.ProtobufLanguage
 
 /**
  * Project level service to keep track of all changes to .proto files, Buf configuration files, and the plugin config.
  * This is used to avoid unnecessary calls to BufAnalyzePass (i.e. 'buf breaking' and 'buf lint').
  */
 @Service(Service.Level.PROJECT)
 class BufAnalyzeModificationTracker(private val project: Project) : ModificationTracker {
-    private val protoModificationTracker = PsiModificationTracker.getInstance(project).forLanguage(PbLanguage.INSTANCE)
+    private val protoModificationTracker = PsiModificationTracker.getInstance(project).forLanguage(protobufLanguage()!!)
```

Comment:
I am not familiar with Kotlin, but I assume this a forcefully unwrapping the optional and will hard crash if neither is present. Can you figure out how to either ensure one is always defined or gracefully handling this situation?